csp/pull.csp

```diff
@@ -9,6 +9,18 @@
 <link rel="stylesheet" type="text/css" href="css/git-webui.css" />
 </head>
 <body>
+<pre for="contextSelect" style="padding-top: 2em;white-space: pre-wrap;height: 60px;">Change Context:  <select id="newContext" onchange="updateContext()">
+	<server>
+		set $NAMESPACE = %request.Data("$NAMESPACE",1)
```

Comment:
Should only change namespace if %request.Data("$NAMESPACE") is defined (just in case people have the old links bookmarked still, this will break badly)